ts/components/menu/items/MenuItemWithDataTestId.tsx

```diff
@@ -0,0 +1,9 @@
+import { Item, ItemProps } from 'react-contexify';
+
+export function ItemWithDataTestId({ children, ...props }: ItemProps) {
+  return (
+    <Item data-testid="context-menu-item" {...props}>
```

Comment:
@Bilb If all the items have the same datatestid will you have to check the text as well in the tests to be able to identity the correct item?

Reply:
yes. we can either 
- get all the `context-menu-item` visible and iterate over each of them
- or ask for a component having data-testid of `context-menu-item` and a text of `plop`